pipeline/minimmit.md

```diff
@@ -205,26 +205,28 @@ _If you have already broadcast `notarize(c, v)` for a `c` that cannot be finaliz
+    - A Byzantine leader can delay the network by at most `4Δ`. This can happen if the leader equivocates by sending different proposals to different replicas, causing them to broadcast `notarize(*, v)` for different blocks. To see why, again assume all honest replicas enter view `v` by time `t`. The first round of `notarize` messages are sent and received by `t + 3Δ`. At this point, all honest replicas observe the conflicting votes and broadcast `nullify(v)`. This "second-round" vote is received by `t + 4Δ`, allowing a `nullification(v)` to be created, allowing them to enter `v+1`.
+    - These latency bounds are competitive despite the possibility of two rounds of voting. This is due to the timer being set to `2Δ`. For comparison, a protocol like Simplex with a `3Δ` timer would see replicas advance to the next view by `t + 4Δ` regardless of whether the leader is crash-faulty or Byzantine.
+- Every view `v` eventually produces at least one certificate `proof(v)`. As soon as a replica sees any `proof(v)` (as fast as `L` messages), it moves to view `v + 1`. If the network is partitioned in two, replicas in each half of the partition may continue to enter successive views (on different `proof(v)`s) but will never be able to finalize such blocks. To bound the depth of forks in a partition, replicas can wait to enter some view `v + k` until they have seen `Q` messages in view `v`.
 
 ## 10. Extensions
 
 Minimmit can be instantiated in several different ways to tune performance when deployed to production. Some examples are below:
 
 - Use block digests (i.e. `c = hash(block)`) in `propose(c, v, (c', v'))`, `notarize(c, v)`, and `notarization(c, v)` messages.
 - Employ BLS multi-signatures or BLS threshold signatures, like [Threshold Simplex](#threshold), to cap `notarization(c, v)` and `nullification(v)` messages at a constant size regardless of the number of replicas.
-- As another optimization taken from [Threshold Simplex](#threshold), for leaders that have been observed to be offline (for some period of time) or who are not directly connected (via p2p), replicas may choose to instantly broadcast `nullify(v)` upon entering view `v` rather than waiting for the timer. This reduces the effect of faulty or crashed nodes on the throughput of the network. Once the faulty replicas reconnect or show "signs of life" via notarizing other proposals, honest replicas can stop "fast-skipping" their views.
+- Broadcast `nullify(v)` when entering the view of a leader that has been offline for some number of recent views, like [Threshold Simplex](#threshold). This reduces the effect of faulty or crashed nodes on a network's block production rate. Once the faulty replicas begin voting in consensus again, honest replicas can stop "fast-skipping" their views.
```

Comment:
nit: "participating in consensus again" (I don't recall if we call `nullify` a vote)